### PR TITLE
Laravel Lightning Talk

### PR DESCRIPTION
**Pull request by cfrome77**
Add documentation for my lightning talk on Laravel, and a folder for images used in the presentation

### CODE REVIEW

File: Laravel.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+}
+
+?>
+```
+
+## How to Setup and Install
+
+- First we needs to install virtual box and vagrant:
+
+  [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
```
Comment (kylelaker) [inline Laravel.md]: Minor suggestion: you can do multi-level bullets with:
```
* First we need to install VirtualBox and Vagrant
    * [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
    * [Vagrant](https://www.vagrantup.com/downloads.html)
```

File: Laravel.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    return view('farms.index', ['farms' => $farms]);
+  }
+}
+
+?>
+```
+
+## How to Setup and Install
+
+- First we needs to install virtual box and vagrant:
```
Comment (kylelaker) [inline Laravel.md]: `s/needs/need`

File: Laravel.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+  and
+
+  [Vagrant](https://www.vagrantup.com/downloads.html)
+
+
+- Once these two are installed type the following in your terminal:
+
+```
+  vagrant box add laravel/homestead
```
Comment (kylelaker) [inline Laravel.md]: Potentially consider left-justifying this section, similar to the `git clone` line (line no. 92)

**Comment by cfrome77** [comment]
changes pushed per @kylelaker's suggestion

**Pull request merged**